index.html

```diff
@@ -236,28 +236,71 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity for verifiable
-credentials using a number of concrete serializations for each of the defined
-syntaxes. The specific set of concrete serializations included will be
-determined by the Working Group.
+This family of specifications consists of documents that each define how to
+express proofs of integrity for verifiable credentials using a number of
+concrete serializations for each of the defined syntaxes. The specific set of
+concrete serializations included will be determined by the Working Group. The
+following are a non-exhaustive selection of expected input documents:
               </p>
-              <p class="draft-status"><b>Draft state:</b> No Draft </p>
-
-              <p class="input-documents"><b>Input documents:</b></p>
-                <p>(the following are a non-exhaustive selection of expected input documents)</p>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a>
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
+              </p>
+              <p class="input-documents"><b>Cryptosuites:</b>
+                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>,
+                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">EdDSA</a>,
+                <a href="https://w3c-ccg.github.io/di-ecdsa-secpr1-2019/">NIST ECDSA</a>,
+                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Koblitz ECDSA</a>
+              </p>
               <p class="milestone"><b>Expected completion:</b> <span class=todo>WG-START+24 months</span></p>
             </dd>
           </dl>
 
         </section>
 
+        <section id="normative">
+          <h3>
+            Conditional Normative Specifications
```

Comment:
We would need to define how these non-W3C document will be used with data-integrity, they on its own do not do that:
```suggestion
            Conditional Normative Features
```

Reply:
VC-JWP isn't a feature, it's a full blown specification. The same could be argued for the other two cryptosuites. Cryptosuites being an extension mechanism and thus typically contained in a separate specification. I can see how we could view these as "Features", but I'm concerned that by doing so, some might argue that they need to be shoved into "existing specifications" or "they don't fit into any of the specifications we're working on". Given that it's the right of the WG to format all of their specifications (combine/split) in whatever way we feel is necessary, I'd rather call these "specifications" to give us maximum flexibility to decide what to do with them in the WG.

For that reason, I'm a -1 on applying this change request.